cloudstack/ProjectService.go

```diff
@@ -1072,6 +1072,9 @@ func (p *UpdateProjectParams) toURLValues() url.Values {
 	if v, found := p.p["account"]; found {
 		u.Set("account", v.(string))
 	}
+	if v, found := p.p["domainid"]; found {
+		u.Set("domainid", v.(string))
+	}
```

Comment:
This is not a valid option when updating a project (see the [docs](https://cloudstack.apache.org/api/apidocs-4.12/apis/updateProject.html))